pyoptsparse/pyParOpt/ParOpt.py

```diff
@@ -2,11 +2,29 @@
 import os
 import datetime
 
-try:
-    from paropt import ParOpt as _ParOpt
-    from mpi4py import MPI
-except ImportError:
-    _ParOpt = None
+# Attempt to import mpi4py.
+# If PYOPTSPARSE_REQUIRE_MPI is set to a recognized positive value, attempt import
+# and raise exception on failure. If set to anything else, no import is attempted.
+if 'PYOPTSPARSE_REQUIRE_MPI' in os.environ:
+    if os.environ['PYOPTSPARSE_REQUIRE_MPI'].lower() in ['always', '1', 'true', 'yes']:
+        from paropt import ParOpt as _ParOpt
+        from mpi4py import MPI
+    else:
+        _ParOpt = None
+# If PYOPTSPARSE_REQUIRE_MPI is unset, attempt to import mpi4py, but continue on failure
+# with a notification.
+else:
+    try:
+        from paropt import ParOpt as _ParOpt
+        from mpi4py import MPI
+    except ImportError:
+        _ParOpt = None
+
+# try:
```

Comment:
Please remove the comment block here.

Reply:
Sorry about leaving the comment in. Removed. 

I will also add a test.

I can install black. Not an issue at all.

Reply:
I pushed up a new version with black formatting and also a test